autoemulate/experimental/emulators/gaussian_process/exact.py

```diff
@@ -46,6 +47,11 @@ class GaussianProcessExact(
 
     """
 
+    # TODO: refactor to work more like PyTorchBackend once any subclasses implemented
```

Comment:
Since this is supposed to be a base class but also the only implementation we have so far I have explicitly defined the optmizer and scheduler in the `__init__` below. Does this make sense and should I set the scheduler to `None`?